dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/Methods/Methods.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public class Methods
+{
+    public static int UseOfOutputInserted(string connectionString, Student newStudent)
+    {
+        using var dbConnection = new SqlConnection(connectionString);
+
+        dbConnection.Open();
+
+        var lastInsertedId = dbConnection.QuerySingle<int>(
+            $"INSERT INTO Students (Firstname, Surname) " +
+            $"OUTPUT INSERTED.Id " +
+            $"VALUES ('{newStudent.Firstname}', '{newStudent.Surname}'); ",
+            newStudent
```

Comment:
We should be using parameterized queries here

Reply:
Converted to parametrized